js/src/utils/createErrorMessageForRejectedPromises.test.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Internal dependencies
+ */
+import createErrorMessageForRejectedPromises from './createErrorMessageForRejectedPromises';
+import createMessageForMultipleErrors from '.~/utils/createMessageForMultipleErrors';
+
+jest.mock( '.~/utils/createMessageForMultipleErrors', () => jest.fn() );
+
+describe( 'createErrorMessageForRejectedPromises', () => {
+	const successPromise = () => new Promise( ( resolve ) => resolve( 'OK' ) );
+	const rejectedPromise = () =>
+		new Promise( ( r, reject ) => reject( 'Failed' ) );
+
+	afterEach( () => {
+		jest.clearAllMocks();
+	} );
+
+	it( 'No rejected Promises', async () => {
```

Comment:
:nail_care: I'd be a bit more descriptive
```suggestion
	it( 'Called with no rejected promises, calls `createMessageForMultipleErrors` with an empty array and `true`', async () => {
```